packages/xrpl/src/models/ledger/Escrow.ts

```diff
@@ -19,8 +21,8 @@ export default interface Escrow extends BaseLedgerEntry {
    * successful.
    */
   Destination: string
-  /** The amount of XRP, in drops, to be delivered by the held payment. */
```

Comment:
Looks like there are other places in this file where the specific references to XRP should be changed to `amount` or `value` or something.

Reply:
The same is likely true in a lot of files here.

Reply:
this was updated I believe